src/pages/welcomePage/WelcomePage.tsx

```diff
@@ -35,7 +35,7 @@ const WelcomePage = () => {
         <ScheduleUploadModalContent
           onSkip={() => {
             setIsUploadingSchedule(false);
-            history.push(PROFILE_PAGE_ROUTE);
+            history.push(PROFILE_PAGE_ROUTE, { fromWelcome: true });
```

Comment:
Hmm I think instead of setting a query param, we should just invalidate the cache when we perform login and not change any of this

Something like:

```
import { useApolloClient } from '@apollo/client'
import { GetUserQuery, GetUserQueryVariables } from 'generated/graphql'

...

client = useApolloClient()

// On login -> client.resetStore()
// OR try client.evict('userId') or something
```